integrationTests/factory/heartbeatComponents/heartbeatComponents_test.go

```diff
@@ -36,10 +36,10 @@ func TestHeartbeatComponents_Close_ShouldWork(t *testing.T) {
 	require.Nil(t, err)
 	managedStatusCoreComponents, err := nr.CreateManagedStatusCoreComponents(managedCoreComponents)
 	require.Nil(t, err)
-	managedCryptoComponents, err := nr.CreateManagedCryptoComponents(managedCoreComponents)
-	require.Nil(t, err)
 	managedNetworkComponents, err := nr.CreateManagedNetworkComponents(managedCoreComponents, managedStatusCoreComponents)
 	require.Nil(t, err)
+	managedCryptoComponents, err := nr.CreateManagedCryptoComponents(managedCoreComponents, managedNetworkComponents)
```

Comment:
managedNetworkComponents should be closed after managedCryptoComponents

Reply:
done